README.md

```diff
@@ -30,9 +30,19 @@ ServiceAssurance) via the YAML configuration to override the default manifests
 loaded for you.
 
 * prometheus_manifest
+* alertmanager_manifest
 * elasticsearch_secret_manifest
 * interconnect_manifest
 * elasticsearch_manifest
 * smartgateway_metrics_manifest
 * smartgateway_events_manifest
 * servicemonitor_manifest
+
+
+## Local testing ##
```

Comment:
Is there a local testing procedure or just the deps?

Reply:
Ah, the deps were first step for documenting local testing. It's not finished, I can remove it, if you prefer.

Reply:
I was more just curious. I think if there aren't testing steps then I'd just drop this section for now.

Reply:
I've added content here: https://github.com/redhat-service-assurance/service-assurance-operator/issues/30